data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -242,6 +245,14 @@ podman run -d --name ironic-ramdisk-logs \
      --entrypoint /bin/runlogwatch.sh \
      -v $IRONIC_SHARED_VOLUME:/shared:z ${IRONIC_IMAGE}
 
+# Failure to start image-customization results in a very confusing error
+icc_id=$(podman ps --filter name=image-customization --filter status=running --format '{{`{{.ID}}`}}')
+if [ -z "$icc_id" ]; then
+    echo The image-customization controller crashed after start
+    podman logs image-customization || true
```

Comment:
I'm not sure about this one (as potentially could expose unwanted information): in case of bootstrap failure the user couldn't check the logs of the container?

Reply:
I assume that people who have access to journald logs also have access to podman? I also don't think ICC actually logs anything sensitive.

Reply:
Here we don't have control on what the ICC could log, in future (and it could be captured by the journalctl reported in CI jobs in case of bootstrap failure). The echo msg seems clear enough to point whoever is troubleshooting the problem in the icc direction.

Reply:
What threat model do you have in mind? People who can read this log, can also read podman logs. It's pure convenience here.